.changeset/pre.json

```diff
@@ -8,5 +8,11 @@
     "@sumup/eslint-plugin-circuit-ui": "0.0.1",
     "@sumup/icons": "2.22.0"
   },
-  "changesets": []
+  "changesets": [
```

Comment:
out of curiosity: what are those values? and where do they came from?

Reply:
They are the autogenerated names of the changeset files (i.e. Markdown files in the `.changeset` folder). Normally, the files are deleted after their content has been added to the changelog during a release. For prereleases, the files are kept around and listed here until the stable release.